### ISSUE TITLE
[Question] how to force renovate to open a PR quickly in a given repo?

### ISSUE THREAD

**Issue by piotr-s-brainhub** [renovatebot/config-help#170]
### Have you checked the logs? Don't forget to include them if relevant
```
DEBUG: 2 flattened updates found: dmx_base, @dmx/npm-winston-slack
DEBUG: Using group branchName template
DEBUG: Dependency dmx_base is part of group All
DEBUG: Using group branchName template
DEBUG: Dependency @dmx/npm-winston-slack is part of group All
DEBUG: Returning 2 branch(es)
DEBUG: generateBranchConfig(1)(branch="renovate/docker-all")
DEBUG: hasGroupName: true(branch="renovate/docker-all")
DEBUG: groupEligible: false(branch="renovate/docker-all")
DEBUG: useGroupSettings: false(branch="renovate/docker-all")
DEBUG: Compiling branchName: renovate/docker-all(branch="renovate/docker-all")
DEBUG: commitMessage: "[renovate] update all dependencies"(branch="renovate/docker-all")
DEBUG: prTitle: "[renovate] update all dependencies"(branch="renovate/docker-all")
DEBUG: generateBranchConfig(1)(branch="renovate/all")
DEBUG: hasGroupName: true(branch="renovate/all")
DEBUG: groupEligible: false(branch="renovate/all")
DEBUG: useGroupSettings: false(branch="renovate/all")
DEBUG: Compiling branchName: renovate/all(branch="renovate/all")
DEBUG: commitMessage: "[renovate] update all dependencies"(branch="renovate/all")
DEBUG: prTitle: "[renovate] update all dependencies"(branch="renovate/all")
DEBUG: config.repoIsOnboarded=true
INFO: Processing 2 branches: renovate/all, renovate/docker-all
DEBUG: Retrieving PR list
DEBUG: Retrieved 56 Pull Requests
DEBUG: PR hourly limit remaining: 5
DEBUG: processBranch with 1 upgrades(dependencies=["@dmx/npm-winston-slack"], branch="renovate/all")
DEBUG: Retrieving branchList(dependencies=["@dmx/npm-winston-slack"], branch="renovate/all")
DEBUG: branchExists(renovate/all)=false(dependencies=["@dmx/npm-winston-slack"], branch="renovate/all")
DEBUG: getBranchPr(renovate/all)(dependencies=["@dmx/npm-winston-slack"], branch="renovate/all")
DEBUG: findPr(renovate/all, null, open)(dependencies=["@dmx/npm-winston-slack"], branch="renovate/all")
DEBUG: branchExists=false(dependencies=["@dmx/npm-winston-slack"], branch="renovate/all")
DEBUG: Branch has 1 upgrade(s)(dependencies=["@dmx/npm-winston-slack"], branch="renovate/all")
DEBUG: recreateClosed is false(dependencies=["@dmx/npm-winston-slack"], branch="renovate/all")
DEBUG: findPr(renovate/all, [renovate] update all dependencies, !open)(dependencies=["@dmx/npm-winston-slack"], branch="renovate/all")
DEBUG: Found PR #56(dependencies=["@dmx/npm-winston-slack"], branch="renovate/all")
DEBUG: Found closed PR with current title(dependencies=["@dmx/npm-winston-slack"], branch="renovate/all")

...

DEBUG: Closed PR already exists. Skipping branch.(dependencies=["@dmx/npm-winston-slack"], branch="renovate/all")
{
  "prTitle": "[renovate] update all dependencies"
}
INFO: Merged PR is blocking this branch(dependencies=["@dmx/npm-winston-slack"], branch="renovate/all")
{
  "pr": 56
}
DEBUG: processBranch with 1 upgrades(dependencies=["dmx_base"], branch="renovate/docker-all")
DEBUG: branchExists(renovate/docker-all)=false(dependencies=["dmx_base"], branch="renovate/docker-all")
DEBUG: getBranchPr(renovate/docker-all)(dependencies=["dmx_base"], branch="renovate/docker-all")
DEBUG: findPr(renovate/docker-all, null, open)(dependencies=["dmx_base"], branch="renovate/docker-all")
DEBUG: branchExists=false(dependencies=["dmx_base"], branch="renovate/docker-all")
DEBUG: Branch has 1 upgrade(s)(dependencies=["dmx_base"], branch="renovate/docker-all")
DEBUG: recreateClosed is false(dependencies=["dmx_base"], branch="renovate/docker-all")
DEBUG: findPr(renovate/docker-all, [renovate] update all dependencies, !open)(dependencies=["dmx_base"], branch="renovate/docker-all")
DEBUG: Found PR #50(dependencies=["dmx_base"], branch="renovate/docker-all")
DEBUG: Found closed PR with current title(dependencies=["dmx_base"], branch="renovate/docker-all")
DEBUG: Returning from graphql closed PR list(dependencies=["dmx_base"], branch="renovate/docker-all")
DEBUG: Closed PR already exists. Skipping branch.(dependencies=["dmx_base"], branch="renovate/docker-all")
{
  "prTitle": "[renovate] update all dependencies"
}
INFO: Merged PR is blocking this branch(dependencies=["dmx_base"], branch="renovate/docker-all")
{
  "pr": 50
}
```

### What is your question?

I would like renovate to open a PR in https://github.com/dmx-io/npm-logger ASAP. A PR in this repo was already opened and merged about an hour ago but later I published `@dmx/npm-winston-slack` so maybe I have to wait some time. However I have many other repos dependent on https://github.com/dmx-io/npm-logger so I cannot proceed with merging PRs in other repos until renovate opens a PR in https://github.com/dmx-io/npm-logger (theoretically I could merge them but it's waste of time because after opening and merging a PR in https://github.com/dmx-io/npm-logger, I will obtain a bunch of PRs in the same repos).

I was trying to click `If you want to rebase/retry this PR, check this box` in the recently merged PR and to also change its title but these actions don't trigger a new PR.

Also I don't understand what `Merged PR is blocking this branch` means. There is a merged PR but with a previous version of `@dmx/npm-winston-slack`.

**Comment by piotr-s-brainhub** [comment]
I found a solution - updating `renovate.json` even with a minor change like commit message triggers a new PR.

**Comment by rarkins** [comment]
Add `“masterIssue”: true` to your config and Renovate will create an issue within the repo showing all planned and blocked PRs. You should be able to check the blocking PR there to force it to be created. 

The merged PR blocking seems to be a bug because that logic shouldn’t apply to group PRs. Can you copy paste your config for that group?

**Comment by rarkins** [comment]
Here’s an example of what you should see: https://github.com/renovatebot/renovate/issues/2958

**Comment by piotr-s-brainhub** [comment]
@rarkins 

`renovate.json` in https://github.com/dmx-io/npm-logger:
```json
{
  "commitMessage": "[renovate] update all dependencies",
  "groupName": "All",
  "labels": [
    "renovate"
  ],
  "npmToken": "e5fcc145-8ebb-43a8-adab-b9cbdb3c350c",
  "rangeStrategy": "bump",
  "packageRules": [
    {
      "packagePatterns": [
        "^@dmx/"
      ],
      "schedule": "at any time"
    }
  ],
  "schedule": "before 3am on the first day of the month"
}
```
